packages/rancher-webhook/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,10 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -3,6 +3,7 @@
+   catalog.cattle.io/hidden: "true"
+   catalog.cattle.io/namespace: cattle-system
+   catalog.cattle.io/os: linux
++  catalog.cattle.io/permits-os: linux,windows
+   catalog.cattle.io/release-name: rancher-webhook
+ apiVersion: v2
+ appVersion: 0.2.5-rc1
```

Comment:
this might need to be redone? I see we already have an rc2 https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-webhook/1.0.4%2Bup0.2.5-rc2